File: Plugins/Aspose_Tasks_Java_for_Ruby/lib/asposetasksjava/Resources/extendedresourceattributes.rb
module Asposetasksjava
  module ExtendedResourceAttributes
    def initialize()
        data_dir = File.dirname(File.dirname(File.dirname(File.dirname(__FILE__)))) + '/data/'
                
        # Instantiate project object
        project = Rjb::import('com.aspose.tasks.Project').new(data_dir + 'test_tasks.mpp')

		res = project.getResources().toList().get(0)

		ea = Rjb::import('com.aspose.tasks.ExtendedAttribute').new
		ea.setFieldId("11")
		ea.setValue("MyValueEA")
		ea.setValueGuid("MyValueGuidEA")
		res.getExtendedAttributes().add(ea)

		puts "Set extended resource attributes."
    end
  end
end

File: Plugins/Aspose_Tasks_Java_for_PHP/src/aspose/tasks/WorkingWithResources/ExtendedResourceAttributes.php
<?php
namespace Aspose\Tasks\WorkingWithResources;

use com\aspose\tasks\Project as Project;
use com\aspose\tasks\ExtendedAttribute as ExtendedAttribute;

class ExtendedResourceAttributes{

    public static function run($dataDir=null){

        # Instantiate project object
        $project = new Project($dataDir . 'test_tasks.mpp');

		$res = $project->getResources()->toList()->get(0);

		$ea = new ExtendedAttribute();
        $ea->setFieldId("11");
		$ea->setValue("MyValueEA");
		$ea->setValueGuid("MyValueGuidEA");
		$res->getExtendedAttributes()->add($ea);
		print "Set extended resource attributes.".PHP_EOL;
    }
}
?>